rust/otap-dataflow/crates/engine/src/exporter.rs

```diff
@@ -202,10 +450,14 @@ mod tests {
                     }
                     Message::PData(_message) => {
                         self.counter.increment_message();
+                        // Execute optional callback if present
+                        if let Some(callback) = self.test_send_ef {
+                            callback(&effect_handler);
```

Comment:
It would be nice to test calling non-sendable effect handlers as well.

Reply:
It's already tested